src/dtoannotations/Email.java

```diff
@@ -4,9 +4,11 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
+import java.util.regex.Pattern;
 
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.FIELD)
 public @interface Email {
-    //String emailAddress();
+    String emailRegex() default "^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-" +
```

Comment:
If you want to store this String inside the interface, no need to create a default method. You can assign it to the variable. 

String EMAIL_REGEX = ....